doc/aspice_swe3_4/swe_docu_guidelines.md

```diff
@@ -75,6 +75,12 @@ In iceoryx are also custom tags used for extended documentation like errors.
  - `/// @min` , use a type safe implementation if possible!
  - `/// @max` , use a type safe implementation if possible!
 
+### Include Additional Header Into Documentation
+
+In some cases, the implementation of a component is is placed in an internal folder and the header in the public folder acts as a trampoline with some using declarations.
```

Comment:
```suggestion
In some cases, the implementation of a component is placed in an internal folder, and the header in the public folder acts as a trampoline, with some using declarations.
```

Reply:
I'll merge this and fix the typo in the other open PR